### PR TITLE
refactor: deprecate interactive file select, update download docs

### PR DESCRIPTION
**Pull request by bcai2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 <!-- badges: start -->
 
+[![R-CMD-check](https://github.com/trytoolchest/toolchest-client-r/workflows/R-CMD-check/badge.svg)](https://github.com/trytoolchest/toolchest-client-r/actions)
```
Comment (lebovic) [inline README.md:8]: Was adding this back in intentional?

**Review by lebovic — APPROVED** [review]
Two questions on intent, otherwise looks good

File: R/tools.R [hunk 2]
```diff
@@ -48,18 +45,6 @@ bowtie2 <- function(tool_args = "", inputs = NULL, output_path = NULL, database_
...
-  if (is.null(read_one)) {
-    read_one <- .choose.path(file_descriptor = "read one")
-
-    if (is.null(read_two)) {
-      read_two <- .choose.path(is_optional = TRUE, file_descriptor = "read two")
-    } else {
-      read_two <- .validate.inpath(read_two)
-    }
-  } else {
-    read_one <- .validate.inpath(read_one)
```
[the remark this answers is not in the document]
Comment (bcai2) [reply R/tools.R:60]: This is intentional. All `.validate.inpath()` does is allow for choosing the input files interactively if they're not provided.

File: README.md (hunk 1 above)
Comment (bcai2) [reply README.md:8]: Nope, fixing now

**Pull request merged**